README.md

```diff
@@ -23,13 +23,35 @@ $ git clone git@github.com:develodesign/vsf-payment-paypal.git ./vue-storefront/
 ```json
 "paypal": {
   "clientId": "",
+  "addPaypalJsToHead": false,
   "endpoint": {
     "complete": "/api/ext/paypal/complete",
     "setExpressCheckout": "/api/ext/paypal/setExpressCheckout"
   }
 }
 ```
 
+#### Add JS to Head
+If you want to add the JS to head you can use `addPaypalJsToHead`.
+
+When set as `true` - This will defer the beforeRegistration hook and then you can add the below into the mounted lifecycle on your checkout component where you will import the Paypal button.
+
+```js
+mounted () {
+    if (!isServer && window.paypalScriptLoaded === undefined) {
+        const storeView = currentStoreView()
+        const { currencyCode } = storeView.i18n
+        const clientId = config.paypal.clientId
+        const sdkUrl = `https://www.paypal.com/sdk/js?client-id=${clientId}&currency=${currencyCode}&disable-funding=card,credit`
```

Comment:
Pls, @JamieSheward apply fix for skip `mybank,sofort`
`disable-funding=card,credit,mybank,sofort`

Reply:
Hey @dimasch, I have commited a fix for the above, including the latest version of the beforeReg hook.

I've also reversed the context and logic of what I added previously
- Check if config.paypal.addJsToGlobalHead is true
- Both beforeReg hook and component will include the check for window var paypalScriptLoaded (Incase if they have the config value set as true, but also have it added into the checkout component mounted section)